README.md

```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
 data from sources like logs and metrics or IT operations
 analytics and security analytics.
 
-ECS is still under development and backward compatibility is not guaranteed. Any
-feedback on the general structure, missing fields, or existing fields is appreciated.
-For contributions please read the [Contributing Guide](CONTRIBUTING.md).
+## Maturity
+
+With ECS turning 1.0, the team will approach improvements by following SemVer.
```

Comment:
Instead of describing SemVer I would just link to SemVer: https://semver.org/

Reply:
Yeah I'm good with adding the link and stripping out the rest for now. If people constantly ask, I can add the short explainer again.